lib/compilers/ada.js

```diff
@@ -55,7 +55,7 @@ class AdaCompiler extends BaseCompiler {
                 '-eS', // commands are not errors
                 '-cargs', // Compiler Switches for gcc.
                 '-o',
-                outputFilename,
+                outputFilename
```

Comment:
just noticed this - so it seems at least _sometimes_ we have functions which use the trailing comma - maybe we should enable this for server side code only?

Browser support is somewhat sketchy, so we couldn't enable it there yet...

Reply:
As per https://eslint.org/docs/rules/comma-dangle#options-14
> functions should only be enabled when linting ECMAScript 2017 or higher.

So we good to enable it server side. I'd say go for it :)